charts/geoweb-opmet-backend/values.yaml

```diff
@@ -92,11 +92,12 @@ opmet:
       failureThreshold: 1
   nginx:
     name: opmet-nginx
-    registry: registry.gitlab.com/opengeoweb/backend-services/opmet-backend/nginx-opmet-backend
-    OPMET_ENABLE_SSL: "FALSE"
+    registry: registry.gitlab.com/opengeoweb/backend-services/auth-backend/auth-backend
+    version: "v0.1.0"
```

Comment:
The latest version (v0.1.1) of the image can used and then this value should be referenced in the opmet-deployment.yaml line 152 -> "{{ .Values.opmet.nginx.version }}"